### DISCUSSION TITLE
Lesson 3: call to FundMe.getPrice errored: execution reverted: 2:52:53 in the video

### DISCUSSION THREAD

**Discussion by tgberkeley** [smartcontractkit/full-blockchain-solidity-course-py#1525] [Q&A]
I have tried to run the code to get the price in lesson 3 but I get an error. I believe I am on the correct environment with the wallet attached correctly to the Rinkeby network. Any help to resolve this issue would be incredibly helpful.

 ```Solidity
 // SPDX-License-Identifier: MIT
  pragma solidity ^0.6.0;
  
  import "@chainlink/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol";
  contract FundMe {
  
      mapping(address => uint256) public addressToFundAmount;
  
      function fund() public payable {
          addressToFundAmount[msg.sender] += msg.value;
      }
  
      function getVersion() public view returns(uint256) {
          AggregatorV3Interface priceFeed = AggregatorV3Interface(0x8A753747A1Fa494EC906cE90E9f37563A8AF630e);
          return priceFeed.version();
      }
  
      
  
      function getPrice() public view returns(uint256){
          AggregatorV3Interface priceFeed = AggregatorV3Interface(0x8A753747A1Fa494EC906cE90E9f37563A8AF630e);
          (,int256 answer,,,) = priceFeed.latestRoundData();
           // ETH/USD rate in 18 digit 
           return uint256(answer * 10000000000);
      }
  }
```

<img width="1322" alt="Screenshot 2022-06-02 at 14 53 46" src="https://user-images.githubusercontent.com/64492814/171645354-2f8923eb-8fd0-4b27-8780-02315ae0867c.png">

**Comment by cromewar** [comment]
Hello @tgberkeley 
Have you enough ETh on your wallet?

**Reply by tgberkeley** [reply]
I have 0.09 ETH in my wallet. Is this enough?
Also the get version function works perfectly fine, it is just when I try the the get price function that it fails.

**Reply by cromewar** [reply]
Actually view functions are suposed to cost 0 gas, so I'm not sure where the error is, could you please try using the code on the repo and see if there is any difference?

**Reply by tgberkeley** [reply]
Hi @cromewar I have copied the code exactly as in the repo and sadly it still does not work, I am not sure why? 

Any help you could provide would be incredible.

**Reply by cromewar** [reply]
On that scenario I think something is blocking your browser, an extension or the cookies. Could you please try using another browser like firefox.
